### PR TITLE
Add post for personal-allowance

### PR DESCRIPTION
**Pull request by inesfb**

### CODE REVIEW

File: _posts/2021-07-25-personal-allowance.md [hunk 1]
```diff
@@ -0,0 +1,1319 @@
...
+title: Funding UBI by eliminating the personal allowance
+date: 2021-06-21
+tags: [uk,ubi]
+subclass: 'post'
+author: ines
+excerpt: A budget-neutral plan exchanging tax exemption for universal basic income
+class: post-template
+---
+
+<script src="https://cdn.plot.ly/plotly-latest.min.js"></script>
```
Comment (MaxGhenis) [inline _posts/2021-07-25-personal-allowance.md:15]: remove these 3 lines and instead add `useplotly: true` in the front matter

File: _posts/2021-07-25-personal-allowance.md [hunk 2]
```diff
@@ -0,0 +1,1319 @@
...
+<script src="https://requirejs.org/docs/release/2.3.5/minified/require.js"></script>
+<script src="https://ajax.googleapis.com/ajax/libs/jquery/3.5.1/jquery.min.js"></script>
+
+# Replacing the UK personal allowance with a universal basic income
```
Comment (MaxGhenis) [inline _posts/2021-07-25-personal-allowance.md:19]: remove these two lines

File: _posts/2021-07-25-personal-allowance.md [hunk 3]
```diff
@@ -0,0 +1,1319 @@
...
+
+The Liberal Democrats were not the first to connect the personal allowance and UBI. The Green Party has also proposed effectively exchanging the two policies[^1], and several think tanks have modeled UBIs involving reductions or repeals of the personal allowance. While the personal allowance is highly salient— a 2014 poll found that [85% of the public](https://www.ipsos.com/ipsos-mori/en-uk/personal-allowances-rise-most-popular-conference-season-tax-pledges) supported raising it—it's also a large source of potential revenue for the costly proposition of paying every member of society every month.
```
Comment (MaxGhenis) [inline _posts/2021-07-25-personal-allowance.md:23]: footnote outside punctuation (please apply to other footnotes throughout)

File: _posts/2021-07-25-personal-allowance.md [hunk 4]
```diff
@@ -0,0 +1,1319 @@
...
+
+### How the personal allowance works
+
+The personal allowance was [introduced in 1979](http://taxhistory.co.uk/Income%20Tax%20Allowances.htm), and it has grown almost every year since. Liberal Democrats accelerated its growth by [calling](https://www.standard.co.uk/news/politics/libdems-to-let-1-3m-low-earners-avoid-paying-tax-8506838.html) for expanding it from £10,000 in 2015 to £12,500 in 2020; this succeeded, and it now sits at £12,570, where it is [expected to remain until April 2026](https://www.reuters.com/article/uk-health-coronavirus-britain-budget-inc-idUSKBN2AV1LP).
+
+While many countries have a tax-free band like the personal allowance, the UK also makes the more uncommon decision to phase it out for higher earners. Individuals earning above £100,000 lose £1 of personal allowance for every £2 in earnings, phasing out fully for those earning £125,140 per year and creating 62% marginal tax rates for earners in this range.
```
Comment (MaxGhenis) [inline _posts/2021-07-25-personal-allowance.md:33]: more uncommon -> uncommon

File: _posts/2021-07-25-personal-allowance.md [hunk 5]
```diff
@@ -0,0 +1,1319 @@
...
+      $("#graph_graph_8_1").load("{{site.baseurl}}assets/markdown_assets/personal-allowance/graph_8_1.html");
+    });
+  </script>
+</div>
+<div id = "graph_graph_8_1"></div>
+
+<div>
+  <script>
+    $(document).ready(function(){
+      $("#graph_graph_8_2").load("{{site.baseurl}}assets/markdown_assets/personal-allowance/graph_8_2.html");
```
Comment (MaxGhenis) [inline _posts/2021-07-25-personal-allowance.md:262]: title is too long, gets cut off

how about `Effect of personal allowance on household net income by decile`?

File: _posts/2021-07-25-personal-allowance.md [hunk 6]
```diff
@@ -0,0 +1,1319 @@
...
+
+Similarly, the [Resolution Foundation](https://www.resolutionfoundation.org/app/uploads/2014/12/Missing-the-target1.pdf) found that the Liberal Democrats’ plan to increase the personal allowance from £10,600 in 2015 to £12,500 by 2020 gave only around £18 of additional annual income to the bottom 10% of households and £203 to the top 10%.
+### Replacing the personal allowance with UBI
+
+While the personal allowance is only slightly progressive by itself, using revenue from repealing the personal allowance to fund a UBI is highly progressive. Fully repealing the Personal Allowance could fund a weekly UBI of £29 per person, and this lowers the Gini index from 0.386 to 0.369.
+
+Each £2,000 decrease in the personal allowance could fund an increase in the UBI amount of around £110 annually, or just over £2 a week.
+
+<button onclick="show_code_11()">Click to show code</button>
```
Comment (MaxGhenis) [inline _posts/2021-07-25-personal-allowance.md:279]: reverse x axis for consistency

File: _posts/2021-07-25-personal-allowance.md [hunk 7]
```diff
@@ -0,0 +1,1319 @@
...
+<div>
+  <script>
+    $(document).ready(function(){
+      $("#graph_graph_11_1").load("{{site.baseurl}}assets/markdown_assets/personal-allowance/graph_11_1.html");
+    });
+  </script>
+</div>
+<div id = "graph_graph_11_1"></div>
+
+This policy would, in general, serve as an income transfer from the richest 40% to the poorest 60%. You can adjust the personal allowance amount with the slider to see how partial personal allowance replacements fare: with further cuts to the personal allowance and higher universal basic incomes, reforms achieve stronger redistributive effects.
```
Comment (MaxGhenis) [inline _posts/2021-07-25-personal-allowance.md:336]: ensure all decile values are visible on x axis

File: _posts/2021-07-25-personal-allowance.md [hunk 8]
```diff
@@ -0,0 +1,1319 @@
...
+  <script>
+    $(document).ready(function(){
+      $("#graph_graph_31_1").load("{{site.baseurl}}assets/markdown_assets/personal-allowance/graph_31_1.html");
+    });
+  </script>
+</div>
+<div id = "graph_graph_31_1"></div>
+
+#### Impacts for specific household types
+Overall, 52% of people come out ahead, most of them belonging to the bottom four deciles. 91% of the lowest decile, 73% of the second-lowest decile, and 62% of the third-lowest decile would see their incomes rise by over 5%. In contrast, most people who see their income decrease are in the upper deciles and lose less than 5%.
```
Comment (MaxGhenis) [inline _posts/2021-07-25-personal-allowance.md:947]: just a note that using the slider on this chart creates an awesome visual effect, love it

Comment (MaxGhenis) [inline _posts/2021-07-25-personal-allowance.md:947]: Clarifying that this refers to full repeal:
```suggestion
Entirely replacing the personal allowance with UBI would benefit 52% on a net basis, most of them belonging to the bottom four deciles. 91% of the lowest decile, 73% of the second-lowest decile, and 62% of the third-lowest decile would see their incomes rise by over 5%. In contrast, most people who see their income decrease are in the upper deciles, and these people mostly lose less than 5%.
```

File: _posts/2021-07-25-personal-allowance.md [hunk 9]
```diff
@@ -0,0 +1,1319 @@
...
+<div>
+  <script>
+    $(document).ready(function(){
+      $("#graph_graph_35_1").load("{{site.baseurl}}assets/markdown_assets/personal-allowance/graph_35_1.html");
+    });
+  </script>
+</div>
+<div id = "graph_graph_35_1"></div>
+
+A single individual living alone would see their income rise when they make under £17,000 and over £117,000.
```
Comment (MaxGhenis) [inline _posts/2021-07-25-personal-allowance.md:1058]: Hide series other than baseline and reform net income

Comment (MaxGhenis) [inline _posts/2021-07-25-personal-allowance.md:1058]: Slider says reform= instead of PA=

Comment (MaxGhenis) [inline _posts/2021-07-25-personal-allowance.md:1058]: Remove `value` from y axis title, just leave blank by setting to `""`

File: _posts/2021-07-25-personal-allowance.md [hunk 10]
```diff
@@ -0,0 +1,1319 @@
...
... replacing the personal allowance with UBI. In 2019, the New Economics Foundation [modeled](https://neweconomics.org/2019/03/nothing-personal) replacing the personal allowance with a budget-neutral Weekly National Allowance paid to adults and an increase in the child benefit, making the payments non-taxable but included in means tests. They found progressive results, stating it would “[lift] 200,000 families out of poverty and shift the £8bn currently spent on tax allowances for the 35% highest income families to the remaining 65% of families”. Compass also [modeled](https://www.compassonline.org.uk/wp-content/uploads/2019/03/Compass_BasicIncomeForAll_2019.pdf) replacing the personal allowance with a budget-neutral flat payment of £25 a week for children and adults of working age, as well as a flat rate citizen’s pension of £164.35. They too found progressive results, with the Gini coefficient decreasing from 0.337 to 0.365. Most recently, [my colleagues found](https://www.ubicenter.org/progressive-adjustments-lib-dem-working-group) that the policy in the Liberal Democrats discussion paper could be made more progressive by repealing all, rather than part, of the personal allowance.
```
Comment (MaxGhenis) [inline _posts/2021-07-25-personal-allowance.md:1315]: move period inside quotes

File: _posts/2021-07-25-personal-allowance.md [hunk 11]
```diff
@@ -0,0 +1,1319 @@
...
+
+Beyond these other papers, I've found that the impacts occur roughly linearly on the margin, meaning that the policy concept can operate incrementally. Furthermore, the policy disproportionately benefits children, and separate studies show that this will in turn improve educational and health outcomes. The impacts are largely but not uniformly beneficial to low income households, though losses are small.
```
Comment (MaxGhenis) [inline _posts/2021-07-25-personal-allowance.md:1317]: link "separate studies" to child-allowance.ubicenter.org/empirical

File: _posts/2021-07-25-personal-allowance.md [hunk 12]
```diff
@@ -0,0 +1,1319 @@
+---
+layout: post
+current: post
+cover: assets/images/personal-allowance.png
+navigation: True
+title: Funding UBI by eliminating the personal allowance
+date: 2021-06-21
+tags: [uk,ubi]
```
Comment (MaxGhenis) [inline _posts/2021-07-25-personal-allowance.md:8]: it's all `ubi` here ;)
```suggestion
tags: [uk]
```

File: _posts/2021-07-25-personal-allowance.md [hunk 13]
```diff
@@ -0,0 +1,1319 @@
+---
+layout: post
+current: post
+cover: assets/images/personal-allowance.png
+navigation: True
+title: Funding UBI by eliminating the personal allowance
```
Comment (MaxGhenis) [inline _posts/2021-07-25-personal-allowance.md:6]:
```suggestion
title: Funding UBI by eliminating the UK personal allowance
```

File: _posts/2021-07-25-personal-allowance.md [hunk 14]
```diff
@@ -0,0 +1,1319 @@
...
+date: 2021-06-21
+tags: [uk,ubi]
+subclass: 'post'
+author: ines
+excerpt: A budget-neutral plan exchanging tax exemption for universal basic income
```
Comment (MaxGhenis) [inline _posts/2021-07-25-personal-allowance.md:11]:
```suggestion
excerpt: The budget-neutral reform would lower poverty by up to 29%.
```

File: _posts/2021-07-25-personal-allowance.md [hunk 15]
```diff
@@ -0,0 +1,1319 @@
...
+
+Beyond these other papers, I've found that the impacts occur roughly linearly on the margin, meaning that the policy concept can operate incrementally. Furthermore, the policy disproportionately benefits children, and separate studies show that this will in turn improve educational and health outcomes. The impacts are largely but not uniformly beneficial to low income households, though losses are small.
+
+Given these findings, UBI proponents in the UK should take a serious look at whether keeping the personal allowance is worthwhile, provided the benefits of the UBI that could be funded through eliminating it.
```
Comment (MaxGhenis) [inline _posts/2021-07-25-personal-allowance.md:1319]: Feel free to adjust, but in general looking to reduce normativity/prescriptiveness (also considering that our audience extends beyond UBI proponents)
```suggestion
Reforms that broaden the tax base are often regressive on their own, but how the revenue is spent matters. Repealing the personal allowance is one such instance: using the revenue to fund a UBI switches the distributional sign from regressive to highly progressive.
```

**Review by MaxGhenis — CHANGES_REQUESTED** [review]
Thanks @inesfb, great to see all the interactivity here. Beyond the suggestions in comments, could you please:
1. Use the same UBI Center blues throughout (when using blues) - it also looks like the charts with sliders have a UBI Center blue when first loading the chart, but then revert to a default blue after adjusting the slider, not sure if that's a plotly bug?
2. Re-run with 500 pound increments

@nikhilwoodruff could you also suggest how to remove all code blocks, or adjust the transformation code to do so?
